src/main/java/mil/dds/anet/utils/ResourceUtils.java

```diff
@@ -0,0 +1,78 @@
+package mil.dds.anet.utils;
+
+import java.time.Instant;
+import java.util.List;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response.Status;
+import mil.dds.anet.beans.PersonPositionHistory;
+
+public class ResourceUtils {
+  private static final String TIME_CONFLICT_ERROR_TEXT =
+      "At least one of the positions in the history is occupied for the specified period.";
+
+  public static void validateHistoryInput(String uuid,
```

Comment:
Since this contains some complicated logic, unit tests for this method should be added which check all possible combinations.